web/src/helpers/api.ts

```diff
@@ -44,16 +44,20 @@ export function getUserList() {
   return axios.get<ResponseObject<User[]>>("/api/user");
 }
 
+export function getUserNameById(id: number) {
+  return axios.get<ResponseObject<string>>(`/api/username/${id}`);
+}
+
 export function patchUser(userPatch: UserPatch) {
   return axios.patch<ResponseObject<User>>("/api/user/me", userPatch);
 }
 
-export function getMemoList() {
-  return axios.get<ResponseObject<Memo[]>>("/api/memo");
+export function getMemoList(userID?: number) {
```

Comment:
In frontend, the id should be named in `camelCase`. So it should be `userId: UserId`.